### DISCUSSION TITLE
Unable to create exe

### DISCUSSION THREAD

**Discussion by noob-infinity** [flet-dev/flet#1587] [Q&A] [answered]
### Question

Flet command is not working propely. Whenever I run the command I get below error.

```
Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\runpy.py", line 196, in _run_module_as_main

    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Program Files\Python310\Scripts\flet.exe\__main__.py", line 4, in <module>
ModuleNotFoundError: No module named 'flet.flet'
```
Currently installed version of flet is `0.8.1`

If I switch back to flet `0.3.2` and run flet in cmd . I get  below and it's look fine.

```
usage: flet [-h] [--version] {run,pack} ...

positional arguments:
  {run,pack}
    run       Run Flet app
    pack      Package Flet app to a standalone bundle

options:
  -h, --help  show this help message and exit
  --version   show program's version number and exit
```

As my some parts of my project are done in latest version of flet,I cannot switch back to old version and as I mentioned earlier the flet command  is not working in the latest version Hence I am unable to create executable.

I need help on this as my project is kind of stucked.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment, marked as answer]
I'm unable to repro the issue. Try re-installing "flet" package to a user home, not program files.

**Reply by noob-infinity** [reply]
Yes,It worked.I had uninstalled the existing one and reinstalled it.
